examples/ndbc/buoy_met_request.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2018 Siphon Contributors.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+NDBC Buoy Meteorological Data Request
+=====================================
+
+The NDBC keeps a 45-day recent rolling file for each buoy. This examples shows how to access
+the basic meteorological data from a buoy and make a simple plot.
+"""
+
+import matplotlib.pyplot as plt
+
+from siphon.simplewebservice.ndbc import NDBC
+
+####################################################
+# Get a pandas data frame of all of the observations, meteorological data is the default
+# observation set to query.
+df = NDBC.realtime_observations('41002')
+df.head()
+
+####################################################
+# Let's make a simple time series plot to checkout what the data look like.
+fig = plt.figure(figsize=(12, 10))
```

Comment:
I think I prefer this:
```python
fig, ax = plt.subplots(3, 1, figsize=(12, 10))
ax2b = ax[2].twinx()
ax[1].plot(...)
...
```

What do you think?

Reply:
I don't love that personally - it's CS pure, but not how a scientist would implement a plot. I often named my axes how they would be labeled in the figure (a, b, etc)

Reply:
If you want to label them that way, great. I have a pretty intense aversion to using numbers in variable names where an array will do—especially when you can get an array out of an API call. This is literally an example from my Pythonic plotting workshop unit.

You can still accomplish what you want with:
```python
fig, (a, b, c) = plt.subplots(3, 1, figsize=(12, 10))
```
How’s that read to you?

I’m not interested in how a “typical” scientist would do this—I’m interested in implementing best practice.

Reply:
Then we'll need to change how it's done elsewhere in our workshop materials :)